src/renderer/components/dock/create-resource.store.ts

```diff
@@ -82,9 +83,11 @@ export class CreateResourceStore extends DockTabStore<string> {
 export const createResourceStore = new CreateResourceStore();
 
 export function createResourceTab(tabParams: Partial<IDockTab> = {}) {
-  return dockStore.createTab({
+  const tab =  dockStore.createTab({
```

Comment:
unnecessary change?

Reply:
Yes, it is) It is the result of some experiments with monaco models. Fixed